Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/Util.cs

```diff
@@ -1,55 +1,52 @@
 ﻿using System.Text.RegularExpressions;
 
-namespace Frends.SFTP.DownloadFiles.Definitions
+namespace Frends.SFTP.DownloadFiles.Definitions;
+
+/// <summary>
+/// Helper methods for file modifications
+/// </summary>
+internal static class Util
 {
-    /// <summary>
-    /// Helper methods for file modifications
-    /// </summary>
-    internal static class Util
+    internal static string CreateUniqueFile(string toDir)
```

Comment:
You are not using this method, so it can be removed.

Reply:
Removed.